src/java/voldemort/client/protocol/admin/AdminClient.java

```diff
@@ -4426,16 +4436,16 @@ public String fetchStore(int nodeId,
             int asyncId = response.getRequestId();
             try {
                 return rpcOps.waitForCompletion(nodeId, asyncId, timeoutMs, TimeUnit.MILLISECONDS);
-            } catch (VoldemortException ve) {
-                logger.error("Got an exception from rpcOps.waitForCompletion for nodeId " + nodeId + ", asyncId " +
```

Comment:
I believe, It logs only on AsyncOperationException and not on base VoldemortException type. Sorry for not being clear.